driver_wrapper.py

```diff
@@ -109,11 +110,12 @@ def click_button_change(self, locator_buttons, locator_wait):
         """Special function for press 'Change' button"""
         WebDriverWait(self.driver, self.default_timeout).until(EC.visibility_of_element_located(locator_wait))
         buttons = self.get_elements(locator_buttons)
-        change = None
-        for i in buttons:
-            if i.text == 'Change':
-                change = i
-        change.click()
+        # change = None
+        for button in buttons:
+            if button.text == 'Change':
+                button.click()
+                break
+        # change.click()
```

Comment:
please, remove the comment or uncomment it

Reply:
done